test/utils.js

```diff
@@ -33,6 +33,9 @@ const FIREFOX_PREFERENCES = {
   // NECESSARY for all 57+ builds
   "extensions.legacy.enabled": true,
 
+  // Include log output in browser console
+  "shield.testing.logging.level": 10, // Trace
```

Comment:
I'm actually in favor of a pref in `utils.js`, since then we wouldn't have to remember to turn off logging in `Config.jsm` when shipping the addon. SSAU also pulls the logging level from Config.jsm, so we'd need a patch for it to read the pref instead as well.

Can you update this PR to read the logging level for `initLog()` from this pref in `bootstrap.js` and in any other files like `Feature.jsm` as well?